components/automate-ui/src/app/services/telemetry/telemetry.service.ts

```diff
@@ -233,11 +237,21 @@ export class TelemetryService {
     );
   }
 
-  private emitToPipeline(operation: String, payload: Object) {
+  private emitToPipeline(operation: String, payload: any) {
```

Comment:
can we define a specific type for payload?

Reply:
payload param is common to accept all types of events from segment. Each event have different set of properties. I removed 'any' type change and keep previous code.